### PR TITLE
#122, 115

### PR DESCRIPTION
**Pull request by vitaly-savin**
Created theHire us! page, customer form, added validation. Fixed some ESLint warnings

### CODE REVIEW

File: utils/myUtils.js
Comment (leresolu) [inline utils/myUtils.js:1]: Please use UUID npm package instead: https://www.npmjs.com/package/uuid

**Comment by leresolu** [comment]
Are those changes not related to story #70? Why is it being merged in #102?

File: pages/Hire.vue [hunk 1]
```diff
@@ -1,54 +1,120 @@
...
-            <v-radio label="FFBB00" value="#FFBB00"></v-radio>
-            <v-radio label="ABCDEF" value="#ABCDEF"></v-radio>
-            <v-radio label="123456" value="#123456"></v-radio>
-        </v-radio-group>
-        <div class="square" :class="classBindingObject"></div>
-    </div>
+    <v-container
+        class="pa-0"
+        fluid
+        style="
```
Comment (leresolu) [inline pages/Hire.vue]: pleae remove custom styles leftover from wordpress and use vuetify only

File: pages/Hire.vue [hunk 2]
```diff
@@ -1,54 +1,120 @@
...
+            background-repeat: no-repeat;
+            background-position: center top;
+            background-size: cover;
+            background-image: linear-gradient(180deg,rgba(9,9,21,0) 0%,#090915 100%), url(./img/bcground-home.jpg)!important;">
+        <span
+            style="
+            position: absolute;
```
Comment (leresolu) [inline pages/Hire.vue]: same here

File: pages/Hire.vue [hunk 3]
```diff
@@ -1,54 +1,120 @@
...
+                                color: #FFFFFF;
+                                font-weight: 500;">
+                                {{ invitationText }}
+                        </div>
+                    </v-row>
+                    <v-row no-gutters justify="center">
+                        <v-btn 
+                            elevation="2"
+                            href="/members"
+                            style="background-color: #FFBB01; color:rgb(224, 43, 32); font-size:20px; text-shadow: 0.08em 0.08em 0.08em rgba(0,0,0,0.4); letter-spacing:4px; font-weight:700; padding: 24px 20px;">
```
Comment (leresolu) [inline pages/Hire.vue]: same here

File: pages/Hire.vue [hunk 4]
```diff
@@ -1,54 +1,120 @@
...
-        classBindingObject () {
-            return {
-                green: this.color === "Green",
-                red: this.color === "Red",
-                blue: this.color === "Blue",
-                black: true
-            }
+    methods: {
+        customerCreated(newCustomer) {
+            const newId = generateUUID(5)
```
Comment (leresolu) [inline pages/Hire.vue]: use UUID npm package instead

**Review by leresolu — CHANGES_REQUESTED** [review]

File: pages/Hire.vue (hunk 1 above)
Comment (vitaly-savin) [reply pages/Hire.vue]: done

File: pages/Hire.vue (hunk 2 above)
Comment (vitaly-savin) [reply pages/Hire.vue]: done

File: pages/Hire.vue (hunk 3 above)
Comment (vitaly-savin) [reply pages/Hire.vue]: done

File: utils/myUtils.js
Comment (vitaly-savin) [reply utils/myUtils.js:1]: done

File: pages/Hire.vue (hunk 4 above)
Comment (vitaly-savin) [reply pages/Hire.vue]: done

**Pull request merged**